content/account/app/index.textile

```diff
@@ -0,0 +1,97 @@
+
+* Ably events can trigger webhook requests to external services, or external services can send data to Ably using incoming webhooks.
+* Ably enables continuous streaming of events to external services like Amazon Kinesis or Kafka, or can receive inbound streams from external services.
+* Ably supports various integration methods, including HTTP callbacks, cloud functions, and streaming services.
+
+h3. Queues
+
+View, monitor, and adjust Ably "queues:":/account/app/queues
+
+* Queues provide access to manage existing queues, monitor data flow, and adjust settings such as TTL, max length, and region.
+* Users can contact Ably support for assistance with increasing limits or customizing their queue settings.
+
+h3. Notifications
+
+Manage "notifications":/account/app/notifications using Ably. Create, test, and send notifications to specific devices or channels, and enure everything works correctly.
+
+* Before sending push notifications, you must configure credentials for third-party services like FCM (Android), APNS (iOS), and Web Push (browsers) through the Ably dashboard.
+* The push inspector tool helps test and debug push notifications by manually sending notifications and reviewing device, client, and channel registrations.
+* Define the title, body, and optional data for your push notifications and choose to target notifications to specific channels, devices, or clients.
+
+h3. Dev console
+
+Monitor your application's health in realtime using Ably's "dev console":/account/app/console by -- tracking events like connection status changes and channel activity. Connect to channels, publish messages, manage presence, and view event logs for troubleshooting:
+
+* The dev console provides realtime insights into application-wide events like connection status changes, channel activity, and event logs.
+* You can connect to channels, publish messages, and manage presence, tracking interactions within the channel through the console.
+* The event log table displays detailed records of connection events, which can be used for troubleshooting and diagnosing application issues.
+* Interact with Ably channels using command-line tools like cURL.
+
+h3. Settings
+
+Manage key aspects of your application "settings":/account/app/settings, including security, enabling or disabling the app, configuring rules for channels, and setting up protocol support for different SDKs:
+
+* Manage settings like App ID, name, security (TLS), and enable/disable the application, affecting its ability to accept connections and use Ably services.
+* Set rules for specific channels or namespaces, such as message persistence, TLS-only access, push notifications, and server-side batching.
+* Enable compatibility with various protocols like Pusher, PubNub, and MQTT for communication between client libraries and Ably.
+* Manage notifications for channel activity and occupancy to gain realtime insights into channel usage and performance.
```

Comment:
I'm not sure what this one means (I think the text in the dashboard is misleading too)